ciphers/decrypt_caesar_with_chi_squared.py

```diff
@@ -189,7 +189,7 @@ def decrypt_caesar_with_chi_squared(
                     # Get the amount of times the letter occurs in the message
                     occurrences = decrypted_with_shift.count(letter)
 
-                    # Get the excepcted amount of times the letter should appear based
+                    # Get the accepted amount of times the letter should appear based
                     # on letter frequencies
                     expected = frequencies[letter] * occurrences
```

Comment:
Doesn't it say "expected" here?

Reply:
yes, I have made the changes. Thanks for letting me know.